universal-login-contracts/test/fixtures/basicIdentityMasterCopy.js

```diff
@@ -0,0 +1,29 @@
+import ERC1077MasterCopy from '../../build/ERC1077MasterCopy';
+import ERC1077Proxy from '../../build/ERC1077Proxy';
+import MasterCopy from '../../build/MasterCopy';
+import MockToken from '../../build/MockToken';
+import MockContract from '../../build/MockContract';
+import {deployContract} from 'ethereum-waffle';
+import {utils, Contract} from 'ethers';
+import {addressToBytes32} from '../utils';
+import {OPERATION_CALL} from '../../lib/consts';
+import DEFAULT_PAYMENT_OPTIONS from '../../lib/defaultPaymentOptions';
+
+const {parseEther} = utils;
+const {gasPrice, gasLimit} = DEFAULT_PAYMENT_OPTIONS;
+
+export default async function basicIdentityMasterCopy(provider, [, , , , , , , , , wallet]) {
+  const publicKey = addressToBytes32(wallet.address);
```

Comment:
#someotherpr can we now get rid of ERC725 and adressToBytes32 on number of places